src/plugins/plugin-manager.ts

```diff
@@ -78,6 +79,11 @@ export class PluginManager {
     return this.plugins.get(Type.name) as T;
   }
 
+  onStep(step: Step) {
+    this.get(BrowserConsole) && (this.get(BrowserConsole)._currentStep = step);
```

Comment:
Will replace this with EventEmitter at some point if we require more integration between runner and plugins.